obvious-ci.docker/linux64_obvci/Dockerfile

```diff
@@ -11,7 +11,12 @@ RUN yum install -y \
                    patch \
                    tar \
                    wget \
-                   which
+                   which \
+                   libXext \
+                   libXrender \
+                   libSM \
+                   tk \
```

Comment:
Tk is the only one here which I think is questionable. I can let that slide for a while though (maybe we should package it: https://github.com/conda/conda-recipes/blob/master/tcltk).